api/filters/replacement/replacement.go

```diff
@@ -178,3 +178,27 @@ func makeResId(n *yaml.RNode) *resid.ResId {
 		Namespace: n.GetNamespace(),
 	}
 }
+
+func getFieldPath(fieldPath string) []string {
+	var result []string
+	split := strings.Split(fieldPath, ".")
+
+	for i := 0; i < len(split); i = i + 1 {
```

Comment:
`; i++ {`

Reply:
for some reason I assumed you couldn't do this in go :) glad I was mistaken